tests/acceptance.suite.dist.yml

```diff
@@ -37,5 +37,5 @@ modules:
             admin email: 'admin@mydomain.com'      # email Id of the Admin
             language: 'English (United Kingdom)'   # Language in which you want the Application to be Installed
         AcceptanceHelper:
-            repo_folder: '/home/travis/build/redCOMPONENT-COM/redCORE/' # Path to the Extension to be installed via Install from folder
+            repo_folder: '/home/travis/build/redCOMPONENT-COM/redCORE/.dist/redCORE' # Path to the Extension to be installed via Install from folder
```

Comment:
Ah! got it. I missed that line:

```
unzip .dist/redcore-v*.zip -d .dist/redCORE
```

https://github.com/redCOMPONENT-COM/redCORE/pull/503/files#diff-354f30a63fb0907d4ad57269548329e3R19

So, I understand that your purpose with it is to test not only the extension but also the PHING packager. Is that right?